src/workloads/sharding/ReshardCollectionWithIndexes.yml

```diff
@@ -0,0 +1,175 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  This test expects the target cluster is created using ebs snapshot with 1 billion 1KB records 
+  and has 3 shards, named shard-00, shard-01, shard-02. The collection has 10 indexes including 
+  _id index, single-key index and compound index. ReshardCollection should use same-key resharding 
+  and use shardDistribution parameter to reshard into 2 shards then 3 shards. There will be random 
+  CRUD operations during resharding but should run at a very low rate.
+
+  The workload consists of 3 phases:
+    1. Turning off balancer and make the sharded collection exists on only 1 shard.
+    4. Running read and write operations on the collection while it is being resharded to 2 shards.
+    5. Running read and write operations on the collection while it is being resharded to 3 shards.
+    
+
+  The inserted documents have the following form:
+
+      {
+        _id: 10,
```

Comment:
Can you make it more clear what the form is here?  As it is, it looks like the documents are all literally the same aside from the padding.  I'm guessing what you really mean is that _id is an integer, counter is an integer that's unique for all docs, num1 and num2 are (small?) integers, str1 and str2 are three characters strings, etc?

Reply:
Sounds good.